components/icon-buttons/Web/Web-icon-buttons.md

```diff
@@ -8,227 +8,167 @@ path: /components/buttons
+   aria-pressed="true">
+   <svg class="mdc-icon-button__icon">
+     ...
+   </svg>
+   <svg class="mdc-icon-button__icon mdc-icon-button__icon--on">
+     ...
+  </svg>
+</button>
+```
 
-![Toggle button anatomy](assets/toggle_button_anatomy.png)
+### Icon Button Toggle with an Image
 
-1. Container
-1. Icon
+The icon button toggle can be used with `img` tags.
 
-<b>Container attributes</b>
-<br>
+```html
+<button id="star-this-item"
+   class="mdc-icon-button mdc-icon-button--on"
+   aria-label="Unstar this item"
+   aria-pressed="true">
+   <img src="" class="mdc-icon-button__icon"/>
+   <img src="" class="mdc-icon-button__icon mdc-icon-button__icon--on"/>
+</button>
+```
 
-Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Container ink color | Primary color | `mdc-button-ink-color($color)`
+## API
 
-<b>Icon attributes</b>
+### CSS Classes
 
- Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Icon | N/A |
-Icon color | Secondary color | ` mdc-button-icon-color($color)`|
+CSS Class | Description
+--- | ---
+`mdc-icon-button` | Mandatory.
+`mdc-icon-button--on` | This class is applied to the root element and is used to indicate if the icon button toggle is in the "on" state.
+`mdc-icon-button__icon` | This class is applied to each icon element for the icon button toggle.
+`mdc-icon-button__icon--on` | This class is applied to a icon element and is used to indicate the toggle button icon that is represents the "on" icon.
 
-We recommend using [Material Theming](https://material.io/components/buttons/#theming) to apply your customizations across your application.
-
-## Icon button theming example
-
-The following example shows an icon button with Shrine theming:
-
-![Icon button with Shrine theming](assets/.png)
-
-x
-### Theming with Sass variables
-
-Before importing any MDC Web modules, set `$mdc-theme-primary` and
-`mdc-theme-on-primary` in your Sass file:
+### Sass Mixins
 
-```scss
-$mdc-theme-primary: #FEDBD0;
-$mdc-theme-on-primary: #442C2E;
-```
+To customize an icon button's color and properties, you can use the following mixins.
 
-In this case we also want to customize the icon color:
+Mixin | Description
+--- | ---
+`density($density-scale)` | Sets density scale for icon button. Supported density scales range from `-5` to `0`, (`0` being the default).
+`size($size)` | Sets the padding for the icon button based on overall size.
+`icon-size($width, $height, $padding)` | Sets the width, height, font-size and padding for the icon and ripple. `$height` is optional and defaults to `$width`. `$padding` is optional and defaults to `max($width, $height)/2`. `font-size` is set to `max($width, $height)`.
+`ink-color($color)` | Sets the font color and the ripple color to the provided color value.
+`disabled-ink-color($color)` | Sets the font color to the provided color value for a disabled icon button.
 
-```scss
+### `MDCIconButtonToggle` Properties and Methods
```

Comment:
Titles need to follow sentence case: capitalize the first word, lower-case all following words unless they're acronyms or proper nouns.